pkg/server/http.go

```diff
@@ -36,6 +37,22 @@ func WithMiddleware(f func(http.Handler) http.Handler) HTTPOption {
 	}
 }
 
+// WithFiberMiddleware adds Fiber middleware to the HTTP server.
+func WithFiberMiddleware(f fiber.Handler) HTTPOption {
```

Comment:
Or maybe this one should be `WithMiddleware`, and the current `WithMiddleware` changes to `WithHTTPMiddleware`?